bin/runtime/src/lib.rs

```diff
@@ -170,9 +170,25 @@ pub const MAX_BLOCK_SIZE: u32 = 5 * 1024 * 1024;
 parameter_types! {
     pub const Version: RuntimeVersion = VERSION;
     pub const BlockHashCount: BlockNumber = 2400;
-    pub BlockWeights: frame_system::limits::BlockWeights = frame_system::limits::BlockWeights
-        ::with_sensible_defaults(MAX_BLOCK_WEIGHT, NORMAL_DISPATCH_RATIO);
-    pub BlockLength: frame_system::limits::BlockLength = frame_system::limits::BlockLength
+    pub RuntimeBlockWeights: BlockWeights = BlockWeights::builder()
+        .base_block(BlockExecutionWeight::get())
+        .for_class(DispatchClass::all(), |weights| {
+            weights.base_extrinsic = ExtrinsicBaseWeight::get();
+        })
+        .for_class(DispatchClass::Normal, |weights| {
+            weights.max_total = Some(NORMAL_DISPATCH_RATIO * MAX_BLOCK_WEIGHT);
```

Comment:
actually, you could extract normal_weight value as `NORMAL_DISPATCH_RATIO * MAX_BLOCK_WEIGHT` to clean it a little

Reply:
to my horror Rust does not atm support evaluation at compile-time :/ So to implement this something like [lazy_static](https://crates.io/crates/lazy_static) is needed. 
If this start to look like too much I can bring back the `sensible_defaults` implementation?